src/main/kotlin/skytils/skytilsmod/core/Config.kt

```diff
@@ -774,6 +790,14 @@ object Config : Vigilant(File("./config/skytils/config.toml"), "Skytils", sortin
     )
     var changeAllSameColorTerminalSolver = false
 
+    @Property(
+        type = PropertyType.SELECTOR, name = "Change All to Same Color Solver Mode",
+        description = "Changes the display mode of the solver.",
+        category = "Dungeons", subcategory = "Terminal Solvers",
+        options = ["Normal", "Left/Right", "Monkey"]
```

Comment:
can you pick better names for this cause 99% of people won't know what Monkey is

Reply:
hoping chosen names are better